### ISSUE TITLE
audit-argument-checks: unchecked default parameter are undetected

### ISSUE THREAD

**Issue by Sebl29** [dferber90/eslint-plugin-meteor#678]
Hi,
when writing an Meteor API method with [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), the audit-argument-checks rule does not detect it.
```
Meteor.methods({
   'foo.barBadlyChecked'(bar = null) { // bar will *not* be marked
//        check(bar, Match.OneOf(Object, null));
    }
   'foo.barWellChecked'(bar) { // bar will be marked
//        check(bar, Match.OneOf(Object, null));
    }
})
```

Tested with these versions:
"eslint-plugin-meteor": "4.2.1" and "5.0.0" (I was just upgrading while recognizing that).
Cheers, and thx for the nice plugin!

**Comment by dferber90** [comment]
Thanks for the report. Do you wanna try to spin up a PR to fix it? If not, could you open a PR which has a failing test showing the issue?

**Comment by Sebl29** [comment]
I would really like to (get to known how to write/extend an eslint plugin), but I am sad to say, the next days I am completely out of time. So don't expect a solution soon^^.

**Comment by kylestumpff** [comment]
https://docs.google.com/presentation/d/1KTZ3TH0MNEFkcFOnRI-Uy-NOABK4R7BKnrXeQn4ZIHY/edit?usp=drivesdk

**Comment by dferber90** [comment]
Only a short 2 years later and this is now supported as of v7.2.0 😁
